package.json

```diff
@@ -68,8 +68,8 @@
     "zeebe-bpmn-moddle": "^1.0.0"
   },
   "dependencies": {
-    "@bpmn-io/form-js": "^1.2.0",
-    "@bpmn-io/variable-resolver": "^1.1.0",
-    "bpmn-js": "^14.0.0"
+    "@bpmn-io/form-js": "^1.4.1",
+    "@bpmn-io/variable-resolver": "^1.2.0",
+    "bpmn-js": "^15.2.0"
```

Comment:
Do we want to make form-js and bpmn-js peer dependencies here? I believe this is one of the places where this is granted. 

In fact we don't want to use our own forms + bpmn-js, but resort on whatever is used as part of the stack, right?

Reply:
Agreed, I'll change it

Reply:
==> [7d56c04](https://github.com/bpmn-io/form-variable-provider/pull/4/commits/7d56c046cb4cbf4d92d1645c84b3c9f50e9e1eea)

I also marked `variable-resolver` as peer, since it does not make sense to have form resolving without variable completion

Reply:
Great stuff.